style.css

```diff
@@ -4,64 +4,116 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800&display=swap");
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
 
-@media only screen and (max-device-width: 767px) {
-  /*
-  div#wrapper {
-    width: 400px;
+@media only screen and (device-width <= 767px) {
```

Comment:
- [ ] Looking at the video below, you can see that in some cases, it seems your CSS doesn't apply, this is due to the way your CSS is written. following the rule of mobile-first design, you are not meant to have this media query here. The CSS will just apply to all screens up till the desktop screen where you add a media query for the desktop screens. in your CSS file, in `line 598` the best media rule to use should be `@media (min-width: 768px) {`. I'll suggest looking into this as your Portfolio is meant to look great and responsive across screens. :smile: :+1: :+1: 

- - VIDEO REFERENCE:

https://github.com/ishimwepati/Portfolio/assets/107632780/374c2aa7-e2f8-4783-b111-fd3469fc9360